ss/rocksdb/comparator.go

```diff
@@ -0,0 +1,76 @@
+//go:build rocksdb
+// +build rocksdb
+
+package rocksdb
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/linxGnu/grocksdb"
+)
+
+// CreateTSComparator should behavior identical with RocksDB builtin timestamp comparator.
+// We also use the same builtin comparator name so the builtin tools `ldb`/`sst_dump`
+// can work with the database.
+func CreateTSComparator() *grocksdb.Comparator {
+	return grocksdb.NewComparatorWithTimestamp(
+		"leveldb.BytewiseComparator.u64ts",
```

Comment:
(any issue with using the leveldb name?)

Reply:
Yea this is intentional. Also done in cosmos-sdk https://github.com/cosmos/cosmos-sdk/blob/main/store/storage/rocksdb/comparator.go#L18